src/components/input/input.js

```diff
@@ -110,6 +110,8 @@ function labelDirective() {
  *
  * @param {number=} md-maxlength The maximum number of characters allowed in this input. If this is specified, a character counter will be shown underneath the input.<br/><br/>
  * The purpose of **`md-maxlength`** is exactly to show the max length counter text. If you don't want the counter text and only need "plain" validation, you can use the "simple" `ng-maxlength` or maxlength attributes.
+ * @param {string=} aria-label Aria-label is required when no label is present.  A warning message will be logged in the console if not present.
```

Comment:
This should probably read "`aria-label` is required when no label element is present. `mdInput` will attempt to copy `placeholder` content to `aria-label`. If neither is present, a warning will be logged to the console."